integration-tests-support/splunk/src/test/java/org/apache/camel/quarkus/test/support/splunk/SplunkTestResource.java

```diff
@@ -50,45 +86,141 @@ public Map<String, String> start() {
                     .withEnv("SPLUNK_HEC_TOKEN", HEC_TOKEN)
                     .withEnv("SPLUNK_LICENSE_URI", "Free")
                     .withEnv("TZ", TimeZone.getDefault().getID())
+                    //                    .withLogConsumer(new Slf4jLogConsumer(LOG))
```

Comment:
Do we remove this ? Sometimes, it could be good to have this activated to catch what is happening on ci.

Reply:
I removed the code, as the usage of log consumer is common in tests extensions. (I don't want to keep the logger active, as the docker logs ansible installation of Splunk server, which is quite long)